libraries/pbrlib/pbrlib_defs.mtlx

```diff
@@ -319,19 +319,19 @@
   <nodedef name="ND_mix_bsdf" node="mix" nodegroup="pbr" defaultinput="bg" doc="Mix two BSDF's according to an input mix amount.">
     <input name="fg" type="BSDF" value="" />
     <input name="bg" type="BSDF" value="" />
-    <input name="mix" type="float" value="1.0" uimin="0.0" uimax="1.0" doc="Mixing weight, range [0, 1]." />
+    <input name="mix" type="float" value="0.0" uimin="0.0" uimax="1.0" doc="Mixing weight, range [0, 1]." />
```

Comment:
@niklasharrysson . Another ping to double-check.

Reply:
Yep, this is correct and now consistent with mix of other input types in stdlib.